src/main/java/core/basesyntax/model/Circle.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax.model;
+
+public class Circle extends Figure {
+    private int radius;
+
+    public Circle(String color, int radius) {
+        super(color);
+        this.radius = radius;
+    }
+
+    public int getRadius() {
+        return radius;
+    }
+
+    @Override
+    public double getArea() {
+        return Math.PI * radius * radius;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: " + getClass().getSimpleName()
+                + ", area: " + getArea() + " sq.units, color: "
+                + getColor() + ", radius: " + getRadius());
```

Comment:
```suggestion
                + getColor() + ", radius: " + radius);
```
you have direct access to this field